documentation/examples/quickstart/client/src/main/scala/quickstart/CounterClientFuture.scala

```diff
@@ -3,17 +3,20 @@ package quickstart
 import endpoints4s.xhr
 import endpoints4s.xhr.EndpointsSettings
 
+import scala.concurrent.ExecutionContext
 import scala.concurrent.Future
 
 object CounterClientFuture
     extends CounterEndpoints
     with xhr.future.Endpoints
     with xhr.JsonEntitiesFromSchemas {
 
+  implicit val ec: ExecutionContext = ExecutionContext.global
```

Comment:
Should we use this one instead?

```suggestion
  implicit val ec: ExecutionContext = scala.scalajs.concurrent.JSExecutionContext.queue
```

Or even use scalajs-macrotask-executor?

Reply:
BTW, why is this execution context now necessary?

Reply:
`ExecutionContext`s are leftovers from `FutureLike`. Can be removed.